test/test_onnx.py

```diff
@@ -294,6 +321,33 @@ def test_faster_rcnn(self):
         model(images)
         self.run_model(model, [(images,), (test_images,)])
 
+    # Verify that paste_mask_in_image beahves the same in tracing.
+    # This test also compares both paste_masks_in_image and _onnx_paste_masks_in_image
+    # (since jit_trace witll call _onnx_paste_masks_in_image).
+    def test_paste_mask_in_image(self):
+        masks = torch.rand(10, 1, 26, 26)
+        boxes = torch.rand(10, 4)
+        boxes[:, 2:] += torch.rand(10, 2)
+        boxes *= 50
+        o_im_s = (100, 100)
+        from torchvision.models.detection.roi_heads import paste_masks_in_image
+        from torchvision.models.detection.transform import resize_boxes
+        out = paste_masks_in_image(masks, boxes, o_im_s)
+        jit_trace = torch.jit.trace(paste_masks_in_image,
+                             (masks, boxes,
+                             [torch.tensor(o_im_s[0]),torch.tensor(o_im_s[1])]))
+        out_onnx = jit_trace(masks, boxes, [torch.tensor(100),torch.tensor(100)])
+        assert torch.all(out.eq(out_onnx))
```

Comment:
Can we also add one more test, which runs the `jit_trace` on a different-sized input / boxes, to verify that the control-flow is handled properly?
Something like
```python
assert torch.all(out.eq(out_onnx))

masks = torch.rand(20, 1, 26, 26)
boxes = torch.rand(20, 4)
boxes[:, 2:] += torch.rand(10, 2)
boxes *= 100
o_im_s = (200, 200)

out = paste_masks_in_image(masks, boxes, o_im_s)
out_onnx = jit_trace(masks, boxes, [torch.tensor(200),torch.tensor(200)])
assert torch.all(out.eq(out_onnx))
```

Reply:
I added a separate test but just understood you meant to add it on the same jit_trace which makes more sense. I'll update that

Reply:
Yeah, I meant to reuse the same `jit_trace`, but for different inputs

Reply:
added.
(I actually also realized that o_im_s was traced as a constant after adding the test, and fixed it. thanks!)